cd-cli/cnf-templates/complete-pipeline.yaml

```diff
@@ -353,7 +353,7 @@ Resources:
                   if ( [ "$PROJECT" = "pn-helpdesk-fe" ] ) then
                     sed -i -e "s|pn_helpdesk_fe_commitId=.*|pn_helpdesk_fe_commitId=$COMMIT_ID|" desired-commit-ids-env.sh
                     export PIPELINE_NAME="pn-helpdesk-fe-update-pipeline"
-                  fi                  
+                  fi           
```

Comment:
@AndreaCimini90 in questo file mi aspetto delle modifiche:
- se è necessario mettere il deploy del portale in CD dovrai agire su ChooseAndLaunchPipelineCodeBuild per attivare il progetto giusto a seconda dell'evento ricevuto
- creare un progetto prendendo come rifiermento UpdateFrontendCodeBuildProject che invochi lo script deployShowcaseSite.sh